lib/worker_status.rb

```diff
@@ -0,0 +1,41 @@
+if Rails.configuration.active_job.queue_adapter == :delayed_job
```

Comment:
I was thinking quite a bit about it. My first version looked something like this:
```ruby
module WorkerStatusSidekiq ... end
module WorkerStatusDelayedJob .. end
class WorkerStatus
  extend "WorkerStatus#{Rails.configuration.active_job.queue_adapter.to_s.camelcase}".constantize
end
```
The trade off I see is either code that is evaluated but not used it or a conditional/conditional require.